README.md

```diff
@@ -98,6 +98,16 @@ requires `column`, `curl`, `mail`, and [jq](https://stedolan.github.io/jq/)
    #### Update cromwell server
    * `update-server`
      * Change the cromwell server that new jobs will be submitted to.
+
+   #### Get cost for a workflow
+	 * `cost`
+	   * Get the cost for a workflow.
+		 * Will only work for workflows that completed more than 8 hours ago on GCS.
+		 * Requires the `gcp_bq_cost_table.config` configuration file to exist and contain the big query cost table for your organization.
+	 * `cost-detailed`
```

Comment:
In pythonland I'd like to replace similar commands like this with `cost -v` or something along those lines.

Reply:
Yeah - I can do that here as well, it's just a bit of a PITA so I broke it into 2 separate commands.